src/main/java/org/gbif/api/vocabulary/ContactType.java

```diff
@@ -28,34 +26,114 @@
  * A utility to infer types is provided which has historically been used during data migration activities from legacy
  * systems such as the previous "GBRDS" which was built on MySQL.
  *
- * @see <a href="http://rs.gbif.org/vocabulary/gbif/agent_role.xml">The IPT vocabulary</a>
+ * @see <a href="http://rs.gbif.org/vocabulary/gbif/agent_role.xml">rs.gbif.org vocabulary</a>
  */
 public enum ContactType {
-
+  /**
+   * A contact to contact for further technical information related to the dataset.
+   */
   TECHNICAL_POINT_OF_CONTACT,
+  /**
+   * A contact to contact for further non-technical information related to the dataset.
+   */
   ADMINISTRATIVE_POINT_OF_CONTACT,
+  /**
+   * A contact to contact for further information about the dataset.
+   */
   POINT_OF_CONTACT,
+  /**
+   * A contact who originally gathered/prepared the dataset.
+   */
   ORIGINATOR,
+  /**
+   * A contact responsible for providing the metadata.
+   */
   METADATA_AUTHOR,
+  /**
+   * A primary scientific contact associated with the dataset.
+   */
   PRINCIPAL_INVESTIGATOR,
+  /**
+   * A contact who is an author of a publication that used the dataset, or author of a data paper.
+   */
   AUTHOR,
+  /**
+   * A contact who contributed content to a dataset (the dataset being described may be a composite).
+   */
   CONTENT_PROVIDER,
+  /**
+   * A contact who is responsible for/takes care of the dataset.
+   */
   CUSTODIAN_STEWARD,
+  /**
+   * A contact involved in the publishing/distribution chain of a dataset.
+   */
   DISTRIBUTOR,
+  /**
+   * A contact associated with editing a publication that used the dataset, or a data paper.
+   */
   EDITOR,
+  /**
+   * A contact who owns the dataset (may or may not be the custodian).
+   */
   OWNER,
+  /**
+   * A contact responsible for any post-collection processing of the dataset.
+   */
   PROCESSOR,
+  /**
+   * A contact associated with the publishing of some entity (paper, article, book, etc) based on the dataset, or of a
+   * data paper.
+   */
   PUBLISHER,
+  /**
+   * The contact that makes use of the dataset.
+   */
   USER,
+  /**
+   * The contact providing informatics/programming support related to the dataset.
+   */
   PROGRAMMER,
+  /**
+   * The contact that maintains and documents the specimens in a collection. Some of their duties include preparing and
+   * labeling specimens so they are ready for identification, and protecting the specimens.
+   */
+  CURATOR,
+  /**
+   * A contact who manages the operation of a data system.
+   */
   DATA_ADMINISTRATOR,
+  /**
+   * A contact who manages the operation of a computer system.
+   */
   SYSTEM_ADMINISTRATOR,
+  /**
+   * A contact appointed to lead and represent a Participant's delegation at the GBIF Governing Board.
+   */
   HEAD_OF_DELEGATION,
```

Comment:
Have you considered https://github.com/gbif/directory/blob/master/directory-backend/directory-api/src/main/java/org/gbif/api/vocabulary/directory/GbifRole.java ?

This seems wrong to have 2 enumerations modelling highly similar stuff.

My first impression is that I don't think we should be including these roles in any contact information other than in the Directory WS.